checkov/secrets/runner.py

```diff
@@ -128,6 +128,22 @@ def run(
                     'limit': ENTROPY_KEYWORD_LIMIT
                 }
             ]
+        custom_plugins = os.getenv("CHECKOV_CUSTOM_DETECTOR_PLUGINS_PATH", None)
+        logging.info(f"Custom detector flag set to {custom_plugins}")
+        if custom_plugins:
+            detector_path = f"{custom_plugins}/platform_regex_detector.py"
```

Comment:
```suggestion
            detector_path = f"{custom_plugins}/custom_regex_detector.py"
```
since we talk about custom detectors here, lets also name it like that